input/pagecontent/relationship.md

```diff
@@ -26,7 +25,7 @@ This layering of IGs balances relative adoption and implementation maturity of F
 
```

Comment:
The image is much wider in AU Base compared to AU Core and AU eRequesting. Not wrong but I find AU Core and AU eRequesting easier to look at. Suggest making the image smaller - change style="width:70%".

Reply:
fixed.